### ISSUE TITLE
Change default prefix for 50.00 to FDIS (instead of PRF)

### ISSUE THREAD

**Issue by opoudjis** [metanorma/pubid-iso#191]
I am raising this issue to you, @ronaldtse, because I am presuming that you have been successfully steering development of this gem over the past 6 months, and that this behaviour is intentional.

````
id = Pubid::Iso::Identifier.create(:number=>"1000", :publisher=>"ISO", :year=>2023, :stage=>"50.00")

id.typed_stage_name == "PRF"
id.typed_stage_name != "FDIS"
````

This is independent of `id.to_s(with_prf: true)`, which after all does not impact the type abbreviation.

The same goes for the retrieved stage name: it is

`<stagename abbreviation="PRF IS">Proof of a new International Standard International Standard</stagename>`

not

`<stagename abbreviation="FDIS">Final Draft International Standard</stagename>`

If it is wrong, that is an issue for this gem. I am vaguely aware that whether 60.00 or 50.00 is treated as PRF is ambiguous, and I DO NOT CARE what the resolution of that is. I am pointing out that right now, the abbreviation of 50.00, when queried separately (to populate Presentation XML as cover page metadata) is always PRF, and if different behaviour is desired, it will need to be made very obvious to me how that is achieved.

**Comment by ronaldtse** [comment]
Sorry I don’t really understand your point. it would be helpful rephrase it in a way that can be understood.

**Comment by ronaldtse** [comment]
If you want FDIS, specify it. 50.00 IS NOT FDIS.

**Comment by opoudjis** [comment]
Fine.

For a document with stage = "60.00" or stage = "50.00", the actual stage could be :PRF instead of :FDIS.

@mico, how do I inform pubid-iso that "50.00" is to be interpreted as FDIS instead of PRF, i.e., that the stage is both abbreviation: FDIS and harmonized: "50.00"? They are currently mutually exclusive.

**Comment by ronaldtse** [comment]
@opoudjis ,

> For a document with stage = "60.00" or stage = "50.00", the actual stage could be :PRF instead of :FDIS.

This is incorrect. ISO has stated that FDIS and PRF share all the same harmonized codes.

"60.00" is currently a BUG in what we call the "ISO Typed Stages" scheme. "60.00" (published but not on sale) and "60.60" (published and on sale) are currently undifferentiated, and ISO/CS knows about this problem, and refuses to fix this.

**Comment by ronaldtse** [comment]
@mico the only thing that needs to be done here is to use the "FDIS" prefix as the default rendering for stage "50.00".
